compiler/luci/logex/src/FormattedGraph.cpp

```diff
@@ -2030,6 +2039,21 @@ bool SummaryBuilderLet<SB::STUV>::summary(const luci::CircleSum *node, locop::No
   return use_reducer(tbl(), node, s);
 }
 
+bool SummaryBuilderLet<SB::STUV>::summary(const luci::CircleSVDF *node, locop::NodeSummary &s) const
+{
+  assert(node->fusedActivationFunction() != luci::FusedActFunc::UNDEFINED);
+
+  s.args().append("input", tbl()->lookup(node->input()));
+  s.args().append("weight_feature", tbl()->lookup(node->weight_feature()));
+  s.args().append("weight_time", tbl()->lookup(node->weight_time()));
+  s.args().append("bias", tbl()->lookup(node->bias()));
+  s.args().append("rank", to_str(node->svdf_rank()));
+  s.args().append("asymmetric_quantize_inputs", to_str(node->asymmetric_quantize_inputs()));
+  s.args().append("fused", to_str(node->fusedActivationFunction()));
```

Comment:
why is `input_activation_state` not shown?

Reply:
this is typo, I will add `input_activation_state`